runtime-modules/content/src/lib.rs

```diff
@@ -1426,7 +1446,16 @@ decl_module! {
             Ok(())
         }
 
-        #[weight = 10_000_000] // TODO: adjust Weight
+        /// Withdraw JOY from channel account
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)`
+        /// - DB:
+        ///    - O(1)
+        /// # </weight>
+        #[weight = WeightInfoContent::<T>::withdraw_from_member_channel_balance().max(WeightInfoContent::<T>::withdraw_from_curator_channel_balance())]
```

Comment:
Respecting max. line length:
```suggestion
        #[weight = WeightInfoContent::<T>::withdraw_from_member_channel_balance()
            .max(WeightInfoContent::<T>::withdraw_from_curator_channel_balance())]
```